payload-schemas/schemas/common/repository.schema.json

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
There's an `organization` property here, with a type of `string`
Should be optional, just like the organization object itself.